Lab1.ipynb

```diff
@@ -0,0 +1,482 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x = range(0,20)\n",
+    "mu = 3\n",
+    "prob = stats.poisson.pmf(x, mu) \n",
+    "plt.step(x,prob)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 153,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Non-continuous distributions "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Non-continuous distributions Q1B: Explain the features\n",
+    "Poisson distribution has a discrete probability for each discrete x value. By changing the mean value, mu, the peak of the distribution changes and ultimately the shape of the distribution changes. Poisson distibution is centered around that mu value and the rest of the x values are given a low proability. This is why the distribution peaks at the mean value and goes very flat instantly."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Non-continuous distributions Q1C\n",
+    "Setting my hypothetical value to be 4. \n",
+    "If my signal free data has a poisson distribution, what is the probability that the data produces a signal that is equally or more signal-like than the signal I observed which has a value of 4?\n"
```

Comment:
This one is similar to the comment I left for line 184. I think x = 4 and we observe something about 0.1 from your plot for x = 4?